src/Makefile.test.include

```diff
@@ -99,7 +99,10 @@ BITCOIN_TESTS += \
 endif
 
 test_test_bitcoin_SOURCES = $(BITCOIN_TESTS) $(JSON_TEST_FILES) $(RAW_TEST_FILES)
-test_test_bitcoin_CPPFLAGS = $(AM_CPPFLAGS) -fopenmp $(BITCOIN_INCLUDES) -I$(builddir)/test/ $(TESTDEFS) $(EVENT_CFLAGS)
+test_test_bitcoin_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES) -I$(builddir)/test/ $(TESTDEFS) $(EVENT_CFLAGS)
+if HAVE_OPENMP
+test_test_bitcoin_CPPFLAGS += -fopenmp
```

Comment:
This probably should also have ``-DMULTICORE``, even though I suspect it doesn't matter.

Reply:
yes, missed that. I'll reset the branch and add `-DMULTICORE` but let me know if you want me to preserve flag order